### ISSUE TITLE
Monika dislikes my name...?

### ISSUE THREAD

**Issue by JustNia** [Monika-After-Story/MonikaModDev#9912]
Monika has a problem with my name: Virginia.
 Well, everyone knows how messed up the bullying and dislike for the name can be and is, but Monika just says: "I dont really feel... comfortable calling you that all the time. Could you pick a more... *appropriate* name please?"

Call me ridiculous, but this makes me feel furious, how a person (well, game) that is described the way Monika is described and say that my name makes them feel uncomfortable? why is the word "virginity" or similars still like a taboo for Monika...? i think nobody wants to hear that about their name...
Sorry if this post is ridiculous or useless for the game development, but I wanted to express how I feel about this dialogue and if there is a way to make it less offensive by the ones  who share this name and let Monika actually call us by it without feeling uncomfortable.

**Comment by ashyashton** [comment]
Yeah as dreamscached said, it's not that your name is bad, so try not to be too hurt by it. It's the same way that anyone named "Richard" can't really put the go-to nickname in, or how someone named Tassadar can't use their name because it has "ass" in it (though that one may have been fixed by now)

The names in question aren't actually bad, including yours, they just have a word in them that trips up the detection system. If Monika were able to recognize what your name really is, without that detection issue, then she would absolutely love it, I'm sure. It's a really pretty name and I know she'd agree!

**Comment by Akhor-** [comment]
Oh those family-friendly filters... You can make Monika actually speak the pronouns you have made with your imagination as if they are normal, but the word "virgin" is a big no-no for... I don't know for who, but those who made this filter must be offended by it, somehow. 
PS So did they fix Tassadar? Finally! :)

**Comment by dreamscached** [comment]
Take this attitude elsewhere. The word 'virgin' is definitely not something you'd call a partner on daily basis, as their name.
